test/js-unit/recrasBookingSpec.js

```diff
@@ -185,6 +185,77 @@ describe('RecrasBooking', () => {
         });
     });
 
+    describe('getAvailableDays', () => {
+        beforeEach(() => {
+            this.rb = new RecrasBooking(new RecrasOptions({
+                element: document.createElement('div'),
+                recras_hostname: 'demo.recras.nl',
+            }));
+
+            this.rb.postJson = jasmine.createSpy('postJson').and.callFake(() => new Promise(function(resolve) {
+                resolve();
+            }));
+
+            this.rb.bookingSize = () => 5;
+            this.rb.productCountsBookingSize = () => [
+                {
+                    aantal: this.rb.bookingSize(),
+                    arrangementsregel_id: 42,
+                },
+            ];
+            this.rb.productCountsNoBookingSize = () => [
+                {
+                    aantal: 5,
+                    arrangementsregel_id: 17,
+                },
+                {
+                    aantal: 5,
+                    arrangementsregel_id: 9,
+                },
+                {
+                    aantal: 8,
+                    arrangementsregel_id: 83,
+                },
+            ];
+        });
+
+        it('should work for packages with fixed programme', () => {
```

Comment:
I think these test descriptions could benefit from a more descriptive title, something like 'it should add "boekingsgrootte" parameter for package with "boekingsgrootte" products'